windows-puppet-agent/azuredeploy.parameters.json

```diff
@@ -12,25 +12,16 @@
       "value": "admin password"
```

Comment:
He's essentially doing a rename of the folder windows-puppet-agent to puppet-windows-agent. But we need to keep the windows-puppet-agent folder around with a readme pointing to newer folder for a week or so to let people updated their URLs. So failing windows-puppet-agent is fine. 

@sebastus I think you can go ahead and just remove all template files from windows-puppet-agent and just leave a readme